tests/auth/test_backends.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 from django.contrib.auth.models import User
+from rest_framework.test import APIRequestFactory
 from taskcluster import Auth
-
```

Comment:
If you re-add this blank line, it should fix the isort linter failures on Travis :-)
https://travis-ci.org/mozilla/treeherder/jobs/267810090#L561